packages/backend/src/models/UserModel.ts

```diff
@@ -325,31 +341,16 @@ export class UserModel {
             return user;
         }
         const lightdashUser = mapDbUserDetailsToLightdashUser(user);
+        const projectRoles = await this.getUserProjectRoles(user.user_id);
 
         return {
             userId: user.user_id,
-            ability: defineAbilityForOrganizationMember(lightdashUser),
+            projectRoles,
```

Comment:
I don't think we need to have a separate projectRoles , we can merge the `rules` I was going to do it like this: 
![image](https://user-images.githubusercontent.com/1983672/174773777-49a28f84-17f3-4fb4-aaa4-ecd1201db4c0.png)

This way we only need to do `ability.can` 

Is there any reason to keep them separate ?

Reply:
right, and the frontend can't use the merged ability object built by the backend ? or the frontend needs to check the different rules separately just to show a different forbidden page...

Reply:
> and the frontend can't use the merged ability object built by the backend ?

No since the REST endpoint is going to strip all the methods. Maybe we can send the rules instead, I think that would work. I can give that a try.